data/sql_setup/prepare_schedule_a.sql

```diff
@@ -214,3 +214,16 @@ create trigger nml_sched_a_after_trigger after insert or update
 drop trigger if exists nml_sched_a_before_trigger on disclosure.nml_sched_a;
 create trigger nml_sched_a_before_trigger before delete or update
     on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_delete_update_queues(:START_YEAR_AGGREGATE);
+
+CREATE OR REPLACE FUNCTION insert_sched_master() RETURNS TRIGGER AS $$
+DECLARE
+    child_table_prefix TEXT = TG_ARGV[0];
+    child_table_name TEXT;
+BEGIN
+    child_table_name = child_table_prefix || get_partition_suffix(new.two_year_transaction_period);
+    EXECUTE format('INSERT INTO %I VALUES ($1.*)', child_table_name)
+        USING new;
```

Comment:
Do we want to incorporate the `ON CONFLICT` clause here too to account for updates?  Do you think we'll need a separate `BEFORE UPDATE` trigger instead or in addition to that?